packages/data/setup.cfg

```
@@ -58,7 +58,7 @@ install_requires =
     python-magic
     pysam>=0.21
     rocrate
-    social-auth-core[openidconnect]==4.0.3
+    social-auth-core[openidconnect]>=4.0.3
```

To sync with pyproject.toml:
```suggestion
    social-auth-core>=4.5.0
```